tests/aws/services/cloudformation/resources/test_kinesis.py

```diff
@@ -174,3 +174,17 @@ def test_dynamodb_stream_response_with_cf(deploy_cfn_template, aws_client):
     assert response.get("TableName") == "EventTable"
     assert len(response.get("KinesisDataStreamDestinations")) == 1
     assert "StreamArn" in response.get("KinesisDataStreamDestinations")[0]
+
+
+@markers.aws.unknown
```

Comment:
New tests should be AWS validated

Reply:
On it.

Reply:
I've tested against AWS with @simonrw 's help